cmd/apmsoak/run.go

```diff
@@ -82,6 +91,7 @@ func NewCmdRun() *cobra.Command {
 	cmd.Flags().StringVar(&options.APIKeys, "api-keys", "", "API keys for managed service. Specify key value pairs as `project_id_1:my_api_key,project_id_2:my_key`")
 	cmd.Flags().BoolVar(&options.BypassProxy, "bypass-proxy", false, "Detach from proxy dependency and provide projectID via header. Useful when testing locally")
 	cmd.Flags().StringVar(&options.Loglevel, "log-level", "info", "Specify the log level to use when running this command. Supported values: debug, info, warn, error")
+	cmd.Flags().BoolVar(&options.IgnoreErrors, "ignore-errors", false, "Do not report as a failure HTTP responses with status code different than 200")
```

Comment:
nit: trying to imply that we will stop the program on first non-200 HTTP response
```suggestion
	cmd.Flags().BoolVar(&options.IgnoreErrors, "ignore-errors", false, "Keep running even if non-200 HTTP responses are received")
```